enterprise/micronaut/nbproject/project.xml

```diff
@@ -164,6 +164,15 @@
                         <specification-version>1.76</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.java.lexer</code-name-base>
```

Comment:
Is this dependency used - maybe indirectly ?

Reply:
Pouziva se `JavaTokenId` - viz https://github.com/apache/netbeans/blob/321c99137b2609d1de5fa9bfa576ddaca08fde62/enterprise/micronaut/src/org/netbeans/modules/micronaut/completion/MicronautDataCompletionTask.java#L50